build.sbt

```diff
@@ -74,32 +81,63 @@ lazy val driver = module("driver", "driver/sources")
     libraryDependencies += Dependencies.remoteRobot,
     libraryDependencies += Dependencies.remoteRobotFixtures,
     buildInfoKeys := Seq[BuildInfoKey](version, intellijBuild, "robotVersion" -> Dependencies.remoteRobot.revision),
-    buildInfoPackage := "org.virtuslab.ideprobe"
+    buildInfoPackage := "org.virtuslab.ideprobe",
   )
 
+
+
 lazy val driverTests = testModule("driver-tests", "driver/tests")
   .dependsOn(driver, junitDriver, api % "compile->compile;test->test")
   .usesIdeaPlugin(driverTestPlugin)
 
 lazy val probePlugin = ideaPluginModule("probe-plugin", "probePlugin", publish = true)
   .dependsOn(api)
-  .settings(
-    intellijPluginName := "ideprobe"
-  )
+  .settings(intellijPluginName := "ideprobe")
 
 lazy val driverTestPlugin = ideaPluginModule("probe-test-plugin", "driver/test-plugin")
-  .settings(
-    intellijPluginName := "driver-test-plugin"
-  )
+  .settings(intellijPluginName := "driver-test-plugin")
 
 lazy val junitDriver = module("junit-driver", "driver/bindings/junit")
   .dependsOn(driver, api % "compile->compile;test->test")
   .settings(
-    libraryDependencies ++= Dependencies.junit
+    libraryDependencies ++= Dependencies.junit,
   )
 
+lazy val scalaProbeApi = project(id = "scala-probe-api", path = "extensions/scala/api", publish = true)
+  .dependsOn(api)
+
+lazy val scalaProbePlugin =
+  ideaPluginModule(id = "scala-probe-plugin", path = "extensions/scala/probePlugin", publish = true)
+    .dependsOn(probePlugin, scalaProbeApi)
+    .settings(
+      intellijPluginName := "ideprobe-scala",
+      packageArtifactZipFilter := { file: File =>
+        // We want only this main jar to be packaged, all the library dependencies
+        // are already in the probePlugin which will be available in runtime as we
+        // depend on it in plugin.xml.
+        // The packaging plugin is created to support one plugin per build, so there
+        // seems to be no way to prevent including probePlugin.jar in the dist reasonable way.
+        file.getName == "scala-probe-plugin.jar"
+      },
+      intellijPlugins += "org.intellij.scala:2020.2.753:nightly".toPlugin,
+      name := "scala-probe-plugin",
+    )
+
+lazy val scalaProbeDriver = project(id = "scala-probe-driver", path = "extensions/scala/driver", publish = true)
+  .enablePlugins(BuildInfoPlugin)
+  .disableIdeaPluginDevelopment
+  .dependsOn(scalaProbeApi, junitDriver)
+  .settings(name := "scala-probe-driver")
+
 lazy val scalaTests = testModule("scala-tests", "extensions/scala/tests")
-  .dependsOn(junitDriver)
+  .dependsOn(junitDriver, scalaProbeDriver)
+  .usesIdeaPlugin(scalaProbePlugin)
+
+
+lazy val examples = testModule("examples", "examples")
+  .dependsOn(junitDriver, scalaProbeDriver)
+  .usesIdeaPlugin(scalaProbePlugin)
```

Comment:
this shouldn't be necessary

Reply:
the scala probe plugin is not available on examples' classpath without that

Reply:
this means that something is wrong, it must be available. Imagine someone uses this from different project. Testing should just work by depending on scalaProbeDriver and driver. Take a look why it works for base ideprobe.